Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM diningclub/docker-base-alpine-php7:1.0.11
```

Comment:
This is a private image, so won't work for other people - can we change this to a public PHP 7.1 image?